pisek/__main__.py

```diff
@@ -108,215 +108,215 @@ def add_argument_dataset(parser):
         "--file-contents",
         "-C",
         action="store_true",
-        help="Show file contents on error",
+        help="show file contents on error",
     )
     parser_test.add_argument(
         "--timeout",
         "-t",
         type=float,
-        help="Override time limit for solutions to TIMEOUT seconds.",
+        help="override time limit for solutions to TIMEOUT seconds",
     )
     parser_test.add_argument(
-        "--full", "-f", action="store_true", help="Don't stop on first failure."
+        "--full", "-f", action="store_true", help="don't stop on first failure"
     )
     parser_test.add_argument(
         "--strict",
         action="store_true",
-        help="Interpret warnings as failures (for final check).",
+        help="interpret warnings as failures (for final check)",
     )
     parser_test.add_argument(
         "--repeat-inputs",
         "-n",
         type=int,
         default=1,
-        help="Generate REPEAT_INPUTS times more inputs. (Seeded inputs only)",
+        help="generate REPEAT_INPUTS times more inputs (seeded inputs only)",
     )
     parser_test.add_argument(
         "--all-inputs",
         "-a",
         action="store_true",
-        help="Test each solution on all inputs.",
+        help="test each solution on all inputs",
     )
     parser_test.add_argument(
         "--skip-on-timeout",
         action="store_true",
-        help="Skip all following inputs on first timeout.",
+        help="skip all following inputs on first timeout",
     )
     parser_test.add_argument(
         "--testing-log",
         "-T",
         action="store_true",
-        help="Write test results to testing_log.json.",
+        help="write test results to testing_log.json",
     )
 
     # ------------------------------- pisek clean -------------------------------
 
-    parser_clean = subparsers.add_parser("clean", help="Clean the directory.")
+    parser_clean = subparsers.add_parser("clean", help="clean task directory")
 
     # ------------------------------- pisek config -------------------------------
 
-    parser_config = subparsers.add_parser("config", help="Manage task config.")
+    parser_config = subparsers.add_parser("config", help="manage task config")
     config_subparsers = parser_config.add_subparsers(
-        help="The subcommand to run.", dest="config_subcommand", required=True
+        help="subcommand to run", dest="config_subcommand", required=True
     )
     config_subparsers.add_parser(
-        "update", help="Update config to newest version. (Replaces the config.)"
+        "update", help="update config to newest version (replaces the config)"
     )
 
     # ------------------------------- pisek visualize -------------------------------
 
     parser_visualize = subparsers.add_parser(
-        "visualize", help="Show solutions statistics and closeness to limit."
+        "visualize", help="show solutions statistics and closeness to limit"
```

Comment:
This should probably use the singular form of `solution`.

```suggestion
        "visualize", help="show solution statistics and closeness to limit"
```

Reply:
What? It visualizes multiple solutions.

Reply:
Train tracks also aren't called trains tracks, even though multiple trains run on them.

Reply:
Right, I'm stupid. Should read next time.